### PR TITLE
add request MTU to sample

### PR DESCRIPTION
**Pull request by pawelByszewski**

### CODE REVIEW

File: src/Reducer.js [hunk 1]
```diff
@@ -207,6 +217,7 @@ export function reducer(
 ): ReduxState {
   switch (action.type) {
     case 'LOG':
+      console.debug(action.message)
```
Comment (mikolak) [inline src/Reducer.js:220]: Message, not type?

File: src/SensorTag.js [hunk 2]
```diff
@@ -169,6 +171,12 @@ class SensorTag extends Component<Props, State> {
             title="SIM lose connection"
             style={{flex: 1}}
             onPress={() => this.props.loseConnection()}
+            disabled={!this.isSensorTagReadyToExecuteTests()}
```
Comment (mikolak) [inline src/SensorTag.js]: This overwrites the `SIM lose connection` button, create a separate one.

**Review by mikolak — CHANGES_REQUESTED** [review]

File: src/Reducer.js (hunk 1 above)
Comment (pawelByszewski) [reply src/Reducer.js:220]: I suppose we need something more sophisticated than that, for the time being messages contains all required informations

File: src/SensorTag.js [hunk 3]
```diff
@@ -169,6 +171,11 @@ class SensorTag extends Component<Props, State> {
             title="SIM lose connection"
             style={{flex: 1}}
             onPress={() => this.props.loseConnection()}
+            style={{flex: 1}}
```
Comment (mikolak) [inline src/SensorTag.js]: This still overwrites existing button.

**Review by mikolak — CHANGES_REQUESTED** [review]

**Review by mikolak — APPROVED** [review]

**Pull request merged**